app/code/community/Bolt/Boltpay/Helper/Api.php

```diff
@@ -799,6 +812,7 @@ function ($item) use ($quote, &$calculatedTotal, $boltHelper) {
                         'tax_amount'       => (int) round($shippingAddress->getShippingTaxAmount() * 100),
                         'service'          => $shippingAddress->getShippingDescription(),
                         'carrier'          => $shippingAddress->getShippingMethod(),
+                        'reference'        => $shippingAddress->getShippingMethod(),
```

Comment:
well, if you add reference here, then why ignore https://github.com/BoltApp/bolt-magento1/blob/9996320e72e614cbcc4e9fa3c8dad41d0d4f5429/app/code/community/Bolt/Boltpay/Helper/Api.php#L842

Reply:
@dumega That order creation code is created from a radically different context, which is the admin.  In fact, the reference is critical for the admin to work and is already applied by that time in the code.